qiskit_experiments/curve_analysis/curve_data.py

```diff
@@ -134,3 +135,119 @@ class ParameterRepr:
 
     # Unit
     unit: Optional[str] = None
+
+
+# pylint: disable=invalid-name
```

Comment:
Is this needed?

Reply:
Yes, `p0` is invalid name (lint likes variable name with more than 3 characters).

Reply:
Ah right.